at_cli/lib/src/at_cli.dart

```diff
@@ -42,69 +53,106 @@ class AtCli {
   /// Method to execute verb
   /// input - Commandline arguments and values
   /// return value - verb result
-  Future<dynamic> execute(ArgResults arguments) async {
+  Future<dynamic> execute(String currentAtSign, AtCliPreference atCliPreference,
```

Comment:
better to have a custom object for execute rather than having both cli preference and args